SwanProjects/swanprojects/handlers.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) SWAN Development Team.
+# Author: Omar.Zapata@cern.ch 2021
+import json
+import os
+import shutil
+import subprocess
+
```

Comment:
Every class and function in this Python module and all the other modules below need to be documented with docstrings. See https://numpydoc.readthedocs.io/en/latest/format.html for the format of the docs.

Reply:
Please also check for typos in the current comments.

Reply:
Also please write comments about what the different parts of your code are doing, otherwise it is harder for a reviewer to understand what the code does.